samples/server.c

```diff
@@ -62,7 +62,8 @@ struct server_data {
     size_t bar1_size;
     struct dma_regions regions[NR_DMA_REGIONS];
     struct {
-        uint64_t pending_bytes;
+        uint64_t pending_read;
```

Comment:
Can you explain why we now need two variables?

Reply:
I thought it made it clearer but now I realise it's unnecessary so I'll remove it.